CHANGELOG.md

```diff
@@ -18,3 +18,13 @@
 
 * [BUGFIX] ApplyTheme is now properly exported in index.js
 * [BUGFIX] The default value for `defaultTheme` was moved to fix an error when not defining a theme.
+
+* [CHANGED] React dependency top boundry from `0.14.0-beta1` to `15.x.x`
```

Comment:
@rcline Not sure if any of this is relevant now because I can see you have set the React dependency already, but you might find value in the documentation, I will leave it to you